### DISCUSSION TITLE
App crashing on start after upgrading to Rect Native 0.76

### DISCUSSION THREAD

**Discussion by LA-Johan** [invertase/react-native-firebase#8089] [Q&A] [answered]
I did not have the new architecture enabled before this so might be related to that. Tried reading the documentation but didn't see anything I need to change. 

After I upgrade react native from `0.75.4` to `0.76.0` app crashes on app start on both Android and iOS with 

```
 (NOBRIDGE) ERROR  Warning: Error: No Firebase App '[DEFAULT]' has been created - call firebase.initializeApp()
 ```
 
 I've made no other changes to firebase related code, I'm using versions:

 ```
   "@react-native-firebase/app": "19.0.0",
   "@react-native-firebase/messaging": "19.0.0",
 ```
 
 Does anyone have any pointers?

**Comment by mikehardy** [comment, marked as answer]
Please don't log issues against old versions
You'll want the new version here for the new version there, there were cross-compatibility issues, they are resolved

**Reply by LA-Johan** [reply]
My apologies, didn't realize I wasn't on the latest version, need to look into why my dependency bump script failed here.
